### PR TITLE
Add a linux question about how to list last lines of a file

### PR DESCRIPTION
**Pull request by Shaik-mohd-huzaifa**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -1169,6 +1169,15 @@ const linuxQuiz = [
     Explanation:
       "If you have an empty directory that you would like to remove, then you can use the rmdir command in the terminal.",
     Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/#the-linux-rmdir-command"
+  },
+  {
+    Question: "In Linux, which command is used to list last 10 lines of a file",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1174]:
```suggestion
    Question: "In Linux, which command is used to list the last 10 lines of a file?",
```

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -1169,6 +1169,15 @@ const linuxQuiz = [
...
+    Answer: "tail -10 filename",
+    Distractor1: "bottom -10 filename",
+    Distractor2: "last -10 filename",
+    Distractor3: "filename -10",
+    Explanation: "This Command is used to list last 10 lines of a file",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1179]:
```suggestion
    Explanation: "tail -10 filename is used to list the last 10 lines of a file.",
```

File: src/data/linux-quiz.ts [hunk 3]
```diff
@@ -1169,6 +1169,15 @@ const linuxQuiz = [
...
+    Link: ""
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1180]: The link field cannot be empty

```suggestion
    Link: "https://www.freecodecamp.org/news/the-linux-commands-handbook/"
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Shaik-mohd-huzaifa** [comment]
Applied Changes

**Comment by Shaik-mohd-huzaifa** [comment]
Applied Changes

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**